plugins/plugin-debugger/che-plugin-ext-debugger-client/src/main/java/org/eclipse/che/ide/ext/debugger/client/debug/DebuggerServiceClient.java

```diff
@@ -33,7 +32,13 @@
     /**
      * Attach debugger.
      */
-    Promise<JavaDebuggerInfo> connect(Map<String, String> connectionProperties);
+    Promise<DebuggerInfo> connect(Map<String, String> connectionProperties);
+
+
+    /**
+     * Starts debugger.
+     */
+    Promise<Void> start(@NotNull String id);
```

Comment:
Here i think we need more doc about id, how to manage id to language, technology and  etc.